### DISCUSSION TITLE
Archive issue at project

### DISCUSSION THREAD

**Discussion by hulk510** [community/community#60158] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

I use GitHub projects to manage my todos. I want to hide issue that I don’t do now. 

https://docs.github.com/en/issues/planning-and-tracking-with-projects/managing-items-in-your-project/archiving-items-from-your-project

However I don’t know where archive button in mobile GitHub apps. 
Also how to show archived issues. 

Thanks

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.119.0 (102894307)
Device: iPhone 13 (iOS 16.5.1)
</details>

**Comment by taki-on** [comment, marked as answer]
Thank you for contacting us. Unfortunately, GitHub Mobile does not support archiving project items yet. For now, you can archive project items on the Web. Sorry for the inconvenience and thank you for bearing with us.

**Reply by hulk510** [reply]
I understand.
thank you for a reply!
